yalexs/activity.py

```diff
@@ -137,6 +139,7 @@
     ACTION_LOCK_ONETOUCHLOCK_2,
     ACTION_LOCK_PIN_UNLATCH,
     ACTION_LOCK_PIN_UNLOCK,
+    ACTION_LOCK_FINGER_UNLOCK,
 }
 REMOTE_ACTIONS = {
```

Comment:
ACTIVITY_TO_FIRST_LAST_NAME needs to be added for cases where the user isn't sent

Reply:
I didn't add these new actions to `ACTIVITY_TO_FIRST_LAST_NAME` because I didn't see `ACTION_LOCK_PIN_UNLOCK` added, and the functionality of the action seemed the same. Is it necessary to add these actions to the dictionary? If so, do we also need to add the `ACTION_LOCK_PIN_*` actions to the dictionary?

Reply:
Yes there are rare cases where the api doesn't return a user and we need a fallback.  The Pin unlocks will probably break in this case

Reply:
Makes sense. I have added the fingerprint actions to `ACTIVITY_TO_FIRST_LAST_NAME` map. I will create a separate PR to add the pin actions as well.